filament/src/Renderer.cpp

```diff
@@ -379,6 +359,22 @@ void FRenderer::renderJob(ArenaScope& arena, FView& view) {
     // TODO: the scaling should depends on all passes that need the structure pass
     ppm.structure(fg, pass, svp.width, svp.height, aoOptions.resolution);
 
+    // Apply the TAA jitter to everything after the structure pass, starting with the color pass.
+    if (taaOptions.enabled) {
```

Comment:
@pixelflinger, I read your FIXME comment and I think I did this correctly, but please advise